packer/install-dependencies.sh

```diff
@@ -181,6 +181,11 @@ for PAUSE_VERSION in ${PAUSE_VERSIONS}; do
     CONTAINER_IMAGE="k8s.gcr.io/pause-amd64:${PAUSE_VERSION}"
     pullContainerImage "docker" ${CONTAINER_IMAGE}
     echo "  - ${CONTAINER_IMAGE}" >> ${RELEASE_NOTES_FILEPATH}
+    # Image 'msazurestackdocker/pause-amd64' is the same as 'k8s.gcr.io/pause-amd64'
```

Comment:
how about this instead:

(inside the `for PAUSE_VERSION in ${PAUSE_VERSIONS}; do` loop)
```
for CONTAINER_IMAGE in "k8s.gcr.io/pause-amd64:${PAUSE_VERSION} msazurestackdocker/pause-amd64:${PAUSE_VERSION}"; do
  pullContainerImage "docker" ${CONTAINER_IMAGE}
done
```

Reply:
or

```
for IMAGE_BASE in "k8s.gcr.io/pause-amd64 msazurestackdocker/pause-amd64"; do
  pullContainerImage "docker" ${IMAGE_BASE}/pause-amd64:${CONTAINER_IMAGE}
done
```

Reply:
updated

Reply:
Sorry, my bad, we need to lose the quotes around the image base vals

`for IMAGE_BASE in k8s.gcr.io/pause-amd64 msazurestackdocker/pause-amd64; do`

...